mh/constants.py

```diff
@@ -257,13 +263,13 @@ def slot(item, qty):
 
 # Dummy PAT_BINARY
 PAT_BINARIES = {
-    0x01: {
+    0x01: lambda: {
         "version": 1,
         "content": make_binary_server_type_list(is_jap=IS_JAP)
     },
```

Comment:
I don't think converting the whole object to a lambda returning a dict/object is a nice idea, especially since the `version` field is a constant. Could you please use a lambda or function pointer in the `content` field instead?

Reply:
Fixed this, instead the content is now (usually) just the function definition so it can be called without a lambda. The exception is the first one, where the is_jap parameter is specified.

Reply:
After thinking about this some more, I think having the entire dict be a `lambda`-generated dict will end up being the way to go. This is because, as event quests cycle through, the version is going to change. Version 1 would be for the first set of event quests, version 2 would be for the second set of event quests, etc.

This doesn't pertain to this particular PR so I'll leave it as is, but we're probably going to need to revert this or otherwise dynamically generate the `version` as well as the `content` once a proper event quest cycle is created.